fixtures/build/DeployRevert.json

```diff
@@ -8,7 +8,7 @@
 	],
 	"evm": {
 		"bytecode": {
-			"object": "6080604052348015600f57600080fd5b506000601a57600080fd5b603f8060276000396000f3fe6080604052600080fdfea26469706673582212201cf655fb270fbb5d7d95db40945c9fb07b1f3586c15a7ac8a7f4cc0e022585d164736f6c634300080d0033"
+			"object": "6080604052348015600f57600080fd5b506000601a57600080fd5b603f8060276000396000f3fe6080604052600080fdfea264697066735822122097fd7c0cdcb3a8015df4cce061a4d7f8c2919787475e0b07d1b0ec13223097ac64736f6c63430008100033"
```

Comment:
just one question, why are we updating the bytecodes?

Reply:
I've add one function without parameters into Greeting contract and run compile command. And this command compiles all our contracts.

Reply:
If only `Greeter` contract changes, shouldn't the bytecode of all the other contracts stay the same? Maybe, update only `Greeter` bytecode?